package.json

```diff
@@ -47,7 +49,8 @@
   },
   "type": "module",
   "dependencies": {
-    "dompurify": "^2.4.3"
+    "dompurify": "^2.4.3",
+    "qr-creator": "^1.0.0"
```

Comment:
Does this need a security review?

Reply:
Yes and no (they said it does not need particular review kind of) but in any case, security is aware of it. I asked them before even starting the process and informed them about what was implemented.

Reply:
Ok, that's what matters.